index_io.go

```diff
@@ -12,30 +11,20 @@ import (
 	"unsafe"
 )
 
-// WriteIndex writes an index to a file.
-func WriteIndex(idx Index, filename string) error {
```

Comment:
Let's just leave this and the ReadIndex apis whether they were, despite we not using them.

Reply:
ok